File: csharp/module1/challenges/challenge1.cs
public double? CalculateTip(double theBill, double tip) {
    if (theBill < 0 || tip < 0) {
        Console.Error.WriteLine("Error");
        return null;
    }

    double tipAmount = theBill * (tip / 100);
    return Math.Round(tipAmount, 2);
}

File: javascript/module1/challenges/challenge1.js
function calculateTip(theBill, tip) {
  // check if bill and tip are valid
  if (theBill < 0 || tip < 0) {
    console.error("Error");
    return null;
  }

  let tipAmount = theBill * (tip / 100);
  return tipAmount.toFixed(2); // Rounds the tip to two decimal places
}
